src/temperature.toit

```diff
@@ -13,6 +13,8 @@ interface TemperatureService:
       --minor=0
 
   read -> float
+  close -> none
+
   static READ-INDEX ::= 0
```

Comment:
```suggestion
  read -> float
  static READ-INDEX ::= 0

  close -> none
```

Reply:
follow-up PR moves these lines anyway. keeping as is to avoid conflicts.